gkeys/gkeys/actions.py

```diff
@@ -380,6 +385,11 @@ def speccheck(self, args):
         results = {}
         failed = defaultdict(list)
         self.output('', '\n Checking keys...')
+        '''Login email'''
+        if args.email in ['expiry']:
+            emailer = Emailer(self.config.get_key('login_gentoo'), self.logger)
+            self.logger.debug(_unicode('Emailer started with login: %s') \
+                % _unicode(self.config.get_key('login_gentoo')['login_email']))
```

Comment:
you are still HARD-CODING things...  self.config.get_key('login_gentoo')

I asked you to add a --from option, but --login is a better name.  P L E A S E make it possible to have many logins and select which login to use.